File: app/src/main/java/com/example/cbanda/achieve/presentation/list/TabSectionPageAdapter.java
package com.example.cbanda.achieve.presentation.list;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;


public class TabSectionPageAdapter extends FragmentPagerAdapter {

    public TabSectionPageAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {

        if (position == 0) {
            return new GoalItemFragment();
        } else if (position == 1) {
            return new InspirationFragment();
        } else return null;
    }


    @Override
    public int getCount() {
        return 2;
    }
}

File: app/src/main/java/com/example/cbanda/achieve/presentation/list/TabSectionPageAdapter.kt
package com.example.cbanda.achieve.presentation.list

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

import com.example.cbanda.achieve.presentation.inspiration.InspirationFragment


class TabSectionPageAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment? {

        return when (position) {
            0 -> GoalItemFragment()
            1 -> InspirationFragment()
            else -> null
        }
    }


    override fun getCount(): Int {
        return 2
    }
}




